src/Robot.cpp

```diff
@@ -85,31 +85,41 @@ void Robot::TeleopPeriodic() {
-  SmartDashboard::PutBoolean("Gear Pneumatic false=low", GearPickup::getInstance()->IsGearUp());
-  SmartDashboard::PutBoolean("Gear Squeezer false=open", GearPickup::getInstance()->IsSqueezeOpen());
-  SmartDashboard::PutNumber("Gear Roller EncoderRPM", GearPickup::getInstance()->GetGearIntakeRPM());
-  SmartDashboard::PutNumber("Top Shooter EncoderRPM", Shooter::getInstance()->GetTopShooterRPM());
-  SmartDashboard::PutNumber("Bot Shooter EncoderRPM", Shooter::getInstance()->GetBottomShooterRPM());
-  SmartDashboard::PutNumber("Ball Intake EncoderRPM", Intake::getInstance()->GetIntakeRPM());
-  SmartDashboard::PutNumber("Ball Conveyor EncoderRPM", BallConveyor::getInstance()->GetBallConveyorRPM());
-  SmartDashboard::PutNumber("Lifter OutPut Voltage", Lifter::getInstance()->GetCurrent());
-
 
+  //................Turret SmartDashboard.........
   SmartDashboard::PutNumber("Turret Current", Turret::getInstance()->GetTurretCurrent());
   SmartDashboard::PutNumber("Turret Voltage", Turret::getInstance()->GetTurretVoltage());
   SmartDashboard::PutNumber("Turret Position", Turret::getInstance()->GetTurretPosition());
   SmartDashboard::PutNumber("Turret SetPoint", Turret::getInstance()->GetSetPoint());
 
+  //.................Intake SmartDashboard.......
   SmartDashboard::PutNumber("Intake Current", Intake::getInstance()->GetIntakeCurrent());
+  SmartDashboard::PutNumber("Ball Intake EncoderRPM", Intake::getInstance()->GetIntakeRPM());
+  SmartDashboard::PutNumber("Ball Intake EncoderRPM", Intake::getInstance()->GetIntakeRPM());
 
+  //.................Shooter SmartDashboard.......
   SmartDashboard::PutNumber("Top Shooter Setpoint", Shooter::getInstance()->GetSetPoint());
+  SmartDashboard::PutNumber("Top Shooter EncoderRPM", Shooter::getInstance()->GetTopShooterRPM());
+  SmartDashboard::PutNumber("Bot Shooter EncoderRPM", Shooter::getInstance()->GetBottomShooterRPM());
 
+  //...............GearPickup SmartDashboard......
   SmartDashboard::PutBoolean("Is Gear In", GearPickup::getInstance()->IsGearIn());
+  SmartDashboard::PutBoolean("Gear FwdBanner sensor true=triggered" , GearPickup::getInstance()->IsFwdLimitSwitchClosed());
+  SmartDashboard::PutBoolean("Gear RevBanner sensor true=triggered", GearPickup::getInstance()->IsRevLimitSwitchClosed());
+  SmartDashboard::PutBoolean("Gear Pneumatic false=low", GearPickup::getInstance()->IsGearUp());
+  SmartDashboard::PutBoolean("Gear Squeezer false=open", GearPickup::getInstance()->IsSqueezeOpen());
+  SmartDashboard::PutNumber("Gear Roller EncoderRPM", GearPickup::getInstance()->GetGearIntakeRPM());
+
+  //...............BallConveyor SmartDashboard.....
+  SmartDashboard::PutNumber("Ball Conveyor EncoderRPM", BallConveyor::getInstance()->GetBallConveyorRPM());
+
+  //...............Lifter SmartDashboard.......
+  SmartDashboard::PutNumber("Lifter OutPut Voltage", Lifter::getInstance()->GetCurrent());
```

Comment:
"OutPut" should be "Output", and this appears to be mislabeled. The function is called `GetCurrent()`.